lireincore/imgcache-bundle/0.1/post-install.txt

```diff
@@ -0,0 +1,5 @@
+<bg=blue;fg=white>                                   </>
+<bg=blue;fg=white> Next for LireinCoreImgCacheBundle </>
+<bg=blue;fg=white>                                   </>
+
+  * <fg=blue>Read</> the documentation at <comment>https://github.com/lireincore/LireinCoreImgCacheBundle/blob/master/README.md</>
```

Comment:
I would remove the post install file. We are only using it when there is something to do manually. Someone installing this bundle knows where the doc is.

Reply:
done